src/demo/index.php

```diff
@@ -106,6 +106,7 @@ function gtag() {
                     <label>✕</label>
                     <input class="param inline" type="number" id="height" name="height" alt="Height (px)" placeholder="50" value="50">
                 </span>
+                <input type="button" value="Reset" onclick="preview.reset();">
```

Comment:
Here it looks great, I'd just suggest adding an empty line before it to separate it from the elements above it, for consistency :smile: 

```suggestion

                <input type="button" value="Reset" onclick="preview.reset();">
```